icubam/predicu/analytics_server.py

```diff
@@ -0,0 +1,54 @@
+from pathlib import Path
+
+from absl import logging  # noqa: F401
+
+from .plot import generate_plots
```

Comment:
Do we want to use relative imports vs icubam.predicu.plot?  I'm always weary of python relative imports (but probably because I don't understand/use them well).

Reply:
Implicit relative imports were problematic in Python 2 e.g. `import plot`. In python 3 explicit relative imports are fine e.g. `from . import plot` or `from .plot import generate_plots`. This is done with respect to the current directory and they are non ambiguous: https://mail.python.org/pipermail/python-dev/2010-October/104476.html

But reverted to absolute imports here for now.